rails/spec/system/external_activities_can_be_run_as_offerings_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+RSpec.describe 'External Activities can be run as offerings', type: :system do
+
+  scenario 'External Activity offerings are runnable', js: true do
+    assignable_type = "external activity"
+    portal_class = Portal::Clazz.find_by_name('Class_with_no_assignment')
+    typeName = assignable_type.gsub(/\s/, "_")
+    assignable = typeName.classify.constantize.find_by_name("My Activity")
```

Comment:
This is an example of something that would have to be modified when running using the cypress/playwright gem.